lib/ansible/modules/service.py

```diff
@@ -1096,7 +1110,13 @@ def service_control(self):
         if self.action == "reload":
             self.action = "onereload"
 
-        ret = self.execute_command("%s %s %s %s" % (self.svc_cmd, self.arguments, self.name, self.action))
+        cmd = f"{self.svc_cmd} "
+        # Jail name and verbose take precedence over other arguments
+        if self.arguments.startswith(('-j', '-v')):
+            cmd += f"{self.arguments} {self.name} {self.action}"
+        else:
+            cmd += f"{self.name} {self.action} {self.arguments}"
+        ret = self.execute_command(cmd)
```

Comment:
Since the same pattern appears to be repeated three times, would it make sense to factor this out into a method?